examples/monitors/multiple_monitors_with_shared_payload_template/payload-template-signal

```diff
@@ -0,0 +1,10 @@
+{
+    "scope": "tag:${tag},tag:correlated-signal-example",
```

Comment:
hmm this looks interesting. is this a valid scope value here? Does our UI send a payload that has scope like this?

Reply:
got it. but it's confusing to have on parametric tag and one hardcoded tag here. let's use this to be more clear:
"scope": "tag:prod-us-west"
having multiple tags scenario should be rare IMO and doesn't need to be advertised in the default examples :)

Reply:
Fixed in c83d5eb